Week-2/CSS Advanced/Position the playing cards/cards.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+   <title>Playing cards</title>
```

Comment:
in Head tag --> feel free to add meta data as much as possible for better SEO practices